pkg/reconciler/controller.go

```diff
@@ -18,27 +18,23 @@ package reconciler
 
 import (
 	"context"
+	"knative.dev/pkg/logging"
```

Comment:
This import somehow got separated from other knative imports

Reply:
goland

Reply:
I'm not convinced this is a reason not to make it right.